swe_cp.m

```diff
@@ -489,8 +490,8 @@ function swe_cp(SwE)
     %-Initialise standardised residual images
     %----------------------------------------------------------------------
     % for i = 1:nSres
-    %     VResI(i) = swe_create_vol(sprintf('ResI_%04d.img', i),...
-    %                               DIM, M, sprintf('spm_spm:ResI (%04d)', i),...
+    %     VResI(i) = swe_create_vol(sprintf('ResI_%02d.img', i),...
+    %                               DIM, M, sprintf('spm_spm:ResI (%02d)', i),...
```

Comment:
Sorry to have missed this before... as per https://github.com/NISOx-BDI/SwE-toolbox/issues/9#issuecomment-388373867 this should be `swe_vox_resid_y%04d`